### ISSUE TITLE
Map gets displayed but the markers aren't loaded and there's an error

### ISSUE THREAD

**Issue by carlos-mg89** [mapbox/mapbox-gl-native#11269]
@tobrun I'm experiencing a similar issue than https://github.com/mapbox/mapbox-gl-native/issues/10157 on a Galaxy S6 with at the latest version of Mapbox. I added the renderTextureMode but it didn't fix the error in my case.

This is my activity.xml file:

`<com.mapbox.mapboxsdk.maps.MapView android:id="@+id/mapView" android:layout_width="match_parent" android:layout_height="match_parent" mapbox:mapbox_cameraTargetLat="40.4378693" mapbox:mapbox_cameraTargetLng="-3.8199625" mapbox:mapbox_styleUrl="mapbox://styles/mapbox/outdoors-v10" mapbox:mapbox_cameraZoom="4" mapbox:mapbox_renderTextureMode="true" />`

And this is my build.graddle:

```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 26
    defaultConfig {
        applicationId "me.domain.mypackagedomain"
        minSdkVersion 15
        targetSdkVersion 26
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    implementation 'com.android.support:appcompat-v7:26.1.0'
    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
    implementation 'com.mapbox.mapboxsdk:mapbox-android-sdk:5.4.1'
    implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-cluster-utils:0.1.0'
    implementation "com.mapzen.android:lost:3.0.4"
    compile 'com.google.android.gms:play-services-location:11.8.0'
    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'com.android.support.test:runner:1.0.1'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
}
```

This occurred to me while following the hints to include the cluster marker support: mapbox/mapbox-plugins-android#49

Besides all this, in the logcat I can see the following lines which don't look so good to me (full logcat here [logcat.txt](https://github.com/mapbox/mapbox-gl-native/files/1743783/logcat.txt)):

```
I/art: Thread[1,tid=27970,Native,Thread*=0x75518c7a00,peer=0x76d9ddf8,"main"] recursive attempt to load library "/data/app/me.domain.mypackagedomain-1/lib/arm64/libmapbox-gl.so"
I/art: Thread[1,tid=27970,Native,Thread*=0x75518c7a00,peer=0x76d9ddf8,"main"] recursive attempt to load library "/data/app/me.domain.mypackagedomain-1/lib/arm64/libmapbox-gl.so"
I/art: Thread[1,tid=27970,Native,Thread*=0x75518c7a00,peer=0x76d9ddf8,"main"] recursive attempt to load library "/data/app/me.domain.mypackagedomain-1/lib/arm64/libmapbox-gl.so"
```

Any ideas?

Thanks in advance!

**Comment by tobrun** [comment]
The error log shown isn't an issue, could you add some code how you are adding your markers?

**Comment by carlos-mg89** [comment]
First of all, @tobrun thanks for taking a moment and look into it. Definitely, I explained myself quite horribly and I didn't put the most valuable information. The code. I was a bit overwhelmed at the moment I wrote this issue, and forgot to add it.

I have been digging in the Mapbox documentation and using google as well, and found out this blog post you published a few months ago with a very accurate explanation of how to do what I wanted. Adding thousands of interactive markers. https://blog.mapbox.com/a-guide-to-the-android-symbollayer-api-5daac7b66f2c

Thanks for that post, it really made my day!

I believe we can close this issue, since you manage to reply to it a couple of months ago :)
